lib/tasks/upload_task.rake

```diff
@@ -0,0 +1,232 @@
+      },
+      sender: {
+        login: 'danhalson',
+        id: 2_422_897,
+        node_id: 'MDQ6VXNlcjI0MjI4OTc=',
+        avatar_url: 'https://avatars.githubusercontent.com/u/2422897?v=4',
+        gravatar_id: '',
+        url: 'https://api.github.com/users/danhalson',
+        html_url: 'https://github.com/danhalson',
+        followers_url: 'https://api.github.com/users/danhalson/followers',
+        following_url: 'https://api.github.com/users/danhalson/following{/other_user}',
+        gists_url: 'https://api.github.com/users/danhalson/gists{/gist_id}',
+        starred_url: 'https://api.github.com/users/danhalson/starred{/owner}{/repo}',
+        subscriptions_url: 'https://api.github.com/users/danhalson/subscriptions',
+        organizations_url: 'https://api.github.com/users/danhalson/orgs',
+        repos_url: 'https://api.github.com/users/danhalson/repos',
+        events_url: 'https://api.github.com/users/danhalson/events{/privacy}',
+        received_events_url: 'https://api.github.com/users/danhalson/received_events',
+        type: 'User',
+        site_admin: false
+      },
+      created: false,
+      deleted: false,
+      forced: false,
+      base_ref: nil,
+      compare: 'https://github.com/raspberrypilearning/test-project-dh/compare/4c751b19488a...477420b58228',
+      commits: [
+        {
+          id: '477420b5822859d5b39b907f122ec51d227b2448',
+          tree_id: 'decae51495ca1df2ce4ec322fbc5ff1bcd768528',
+          distinct: true,
+          message: 'Create test.py',
+          timestamp: '2024-09-20T14:14:29+01:00',
+          url: 'https://github.com/raspberrypilearning/test-project-dh/commit/477420b5822859d5b39b907f122ec51d227b2448',
+          author: {
+            name: 'Dan Halson',
+            email: 'danhalson@users.noreply.github.com',
+            username: 'danhalson'
+          },
+          committer: {
+            name: 'GitHub',
+            email: 'noreply@github.com',
+            username: 'web-flow'
+          },
+          added: [
+            'en/code/test.py'
+          ],
+          removed: [],
+          modified: []
+        }
+      ],
+      head_commit: {
+        id: '477420b5822859d5b39b907f122ec51d227b2448',
+        tree_id: 'decae51495ca1df2ce4ec322fbc5ff1bcd768528',
+        distinct: true,
+        message: 'Create test.py',
+        timestamp: '2024-09-20T14:14:29+01:00',
+        url: 'https://github.com/raspberrypilearning/test-project-dh/commit/477420b5822859d5b39b907f122ec51d227b2448',
+        author: {
+          name: 'Dan Halson',
+          email: 'danhalson@users.noreply.github.com',
+          username: 'danhalson'
+        },
+        committer: {
+          name: 'GitHub',
+          email: 'noreply@github.com',
+          username: 'web-flow'
+        },
+        added: [
+          'en/code/test.py'
+        ],
+        removed: [],
+        modified: []
+      }
+    }
```

Comment:
💭 I'd be tempted on pulling this out but it might be more config hassle than its worth

Reply:
sorry got the wrong end of the stick!!